src/coresyftools/coresyftools/tests/test_tester.py

```diff
@@ -1,12 +1,28 @@
 from unittest import TestCase
 
 from ..tool_tester import ToolTester, NonZeroReturnCode, NonEmptyStderr, NoOutputFile, EmptyOutputFile
+import os
 
 
 class TestTester(TestCase):
 
+    def test_download_input(self):
+        tester = ToolTester('coresyftools/tests/dummy_tool',
+                            ('rccc', 'ssdh2dme'))
+        tester._download_input('downloaded_file', "https://scihub.copernicus.eu/dhus/odata/v1/Products('139d30eb-bc03-459c-8642-e794aad61707')/$value", tester.scihub_credentials)
+        self.assertTrue(os.path.exists('downloaded_file'))
+        self.assertGreater(os.path.getsize('downloaded_file'), 0)
+        os.remove('downloaded_file')
+    
+    def test_inputs_mappings(self):
```

Comment:
`test_input_mappings`. Remember to be consistent with singular/plural names, in the `ToolTester` you call them `input_mappings`.

Also, there should be a verb associated. What do you test? That you can define input mappings? That input mappings are correct?

I would suggest `test_can_define_input_mappings` or `test_defines_input_mappings`. (In your defense, I can't think of better names).